py-polars/tests/benchmark/run_h2oai_benchmark.py

```diff
@@ -274,7 +274,7 @@
 out = x.group_by(["id2", "id4"]).agg((pl.corr("v1", "v2") ** 2).alias("r2")).collect()
 print(time.time() - t0)
 assert out.shape == (9216, 3)
-assert np.isclose(out["r2"].sum(), 9.902706276948825)
+assert np.isclose(out["r2"].sum(), 9.86033041315029)
```

Comment:
The online algorithm slightly more numerical error. This is the sum deviation of 10_000 correlations.